logic/vote.js

```diff
@@ -327,10 +327,15 @@ Vote.prototype.schema = {
 			type: 'array',
 			minItems: 1,
 			maxItems: constants.maxVotesPerTransaction,
-			uniqueItems: true
+			uniqueItems: true,
+      items: {
+        type: 'string',
+        pattern: '^[-+]{1}[0-9a-z]{64}$'
```

Comment:
we should also accept [A-Z]

Reply:
Hi @vekexasia I've copied the pattern from logic/vote at line 133. Let me know if you want that I add [A-Z] there too. https://github.com/RiseVision/rise-node/blob/development/logic/vote.js#L133

Reply:
hmm ok. If the whole code does not accept capitalized stuff => keep it that way.